pkg/deployments/tasks/20230206-aave-rebalanced-linear-pool-v4/readme.md

```diff
@@ -0,0 +1,20 @@
+# 2023-02-06 - Aave Rebalanced Linear Pool v4
+
+Deployment of the `AaveLinearPoolFactory`, for Linear Pools with a wrapped aToken.
+
+Supersedes `20221207-aave-rebalanced-linear-pool-v3`, with the following improvements:
+- Adds compatibility with AAVE V3 protocol.
+- Fixes the reentrancy issue described in this [forum post](https://forum.balancer.fi/t/reentrancy-vulnerability-scope-expanded/4345).
```

Comment:
Would you agree with this @joaobrunoah? Anything else I could be missing?

Reply:
@jubeira This pool is actually not compatible with Aave v3. Aave v3 token wrappers will utilize the `ERC4626LinearPool` instead.

Reply:
Thanks; I was pretty sure that was the case but I still added this at the very last minute.
Fixed now.